readme.md

```diff
@@ -94,7 +108,11 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
+A person can get access to a car through a valet key iff they work for the valet
+service.
 
+A person can get access to your site via an OAuth token iff they are registered
+with the third party you are using.
```

Comment:
:+1: Also the idea that valet keys restrict access. When a user approves a 3rd party log-in, you get some but not total access to their account